### PR TITLE
[#IC-47] Admin can change service category and custom_special_flow

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: src/utils/conversions.ts [hunk 1]
```diff
@@ -28,6 +31,23 @@ export const getServicePayloadUpdater = (user: IExtendedUserContract) => (
   const serviceMetadata: ServiceMetadata = {
     ...payload.service_metadata,
     // Scope for visible services cannot be changed
+    category:
+      (SpecialServiceMetadata.is(payload.service_metadata) ||
```
Comment (gunzip) [inline src/utils/conversions.ts:35]: I'd extract these conditions for clarity (ie. `const canSetCustomFlow =` ...)

Comment (BurnedMarshal) [reply src/utils/conversions.ts]: We can reduce code duplication with a method that checks if the category field exists. Conditions are different for the field `category` and `custom_special_flow`. 18855bf
Add some comments in 18855bf

**Review by gunzip — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "build": "npm-run-all -s generate build:clean build:tsc",
     "clean:api-models": "rimraf generated/api && shx mkdir -p generated/api",
     "generate:api-models:public": "gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions/v0.64.0/api/public_api_v1.yaml --no-strict --out-dir generated/api",
-    "generate:api-models:admin": "gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-admin/master/openapi/index.yaml --no-strict --out-dir generated/api",
+    "generate:api-models:admin": "gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-admin/IC-40-special-service-model-upgrade/openapi/index.yaml --no-strict --out-dir generated/api",
```
Comment (balanza) [inline package.json:19]: The PR is fine, but we must not refer to dangling branches

**Review by balanza — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (BurnedMarshal) [reply package.json]: Fixed in acce582

**Pull request merged**